src/Network/GRPC/HighLevel/Generated.hs

```diff
@@ -28,10 +28,17 @@ module Network.GRPC.HighLevel.Generated (
 
   -- * Server Auth
 , ServerSSLConfig(..)
+
+  -- * Client
```

Comment:
Previously, we'd been moving so fast that it felt like even trying to attach a version would be premature. Now might be a good time to start bumping version numbers, though. I'll set it to `0.0.1.0` unless someone has another suggestion.